bcl2fastq/index_summary.py

```diff
@@ -15,10 +15,9 @@
     './data/bcl2fastq_cache.hd5'
 )
 
-all_runs = index['Run'].sort_values(ascending=False).unique()
-
-all_runs = [{'label': r, 'value': r} for r in all_runs]
-
+all_runsog = index['Run'].sort_values(ascending=False).unique()
+print (all_runsog)
```

Comment:
Stray print statement. I suggest using the `git diff` command to check the changes you have introduced before comitting them. This command is also a [PyCharm feature](https://www.jetbrains.com/help/pycharm/comparing-files-and-folders.html)